ga4gh/backend.py

```diff
@@ -141,6 +162,38 @@ def startProfile(self):
     def endProfile(self):
         pass
 
+    def callSetsGenerator(self, request):
+        """
+        Returns a generator over the (callSet, nextPageToken) pairs defined by
+        the specified request.
+        """
+        # if no variantSetIds are input from client,
+        # then set variantSetIds to all variantSetIds
+        if request.variantSetIds == []:
+            variantSetIds = self._variantSetIds
+        else:
+            variantSetIds = request.variantSetIds
+        name = request.name
+
```

Comment:
I don't think we need the blank line here -- we only need extra whitespace in a function when we really need to emphasise two distinct parts of the function. It's pretty clear what's happening here. Likewise below.